src/test/java/core/basesyntax/strategy/BalanceHandlerTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.strategy.impl.BalanceHandler;
+import core.basesyntax.strategy.impl.OperationHandlerStrategyImpl;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    private static Map<String, OperationHandler> operationHandler;
+    private static OperationHandlerStrategyImpl operationHandlerStrategy;
+    private static BalanceHandler balanceStrategy;

```

Comment:
```suggestion
    private static OperationHandler balanceHandler;
```
the same for other implementations test classes